test-base.cfg

```diff
@@ -247,6 +249,53 @@ initialization =
     if len(sys.argv) < 2: sys.argv.extend([pkgdir])
     sys.argv.extend(${pydocstyle:args})
 
+# Downloads the default isort.cfg.
+[isort-cfg]
+recipe = collective.recipe.shelloutput
+cfgfile = ${buildout:directory}/parts/isort/.isort.cfg
+cfgurl = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/.isort.cfg
+commands =
+    cmd1 = mkdir -p `dirname ${isort-cfg:cfgfile}`
+    cmd2 = curl "${isort-cfg:cfgurl}" > "${isort-cfg:cfgfile}"
+
+# bin/isort : isort validation of the packages source.
+[isort]
+recipe = zc.recipe.egg
+
+eggs =
+	isort
+	setuptools
+
+args = [
+    '--recursive',
+    '--check-only',
```

Comment:
I noticed that `-y` ("apply") does not work. I suspect a clash with `check-only` (unverified).
Do we want to address that or ignore for now?